libraries/AP_HAL_ChibiOS/hwdef/MicoAir405Mini/hwdef.dat

```diff
@@ -145,6 +145,9 @@ define OSD_ENABLED 1
 define HAL_OSD_TYPE_DEFAULT 1
 ROMFS_WILDCARD libraries/AP_OSD/fonts/font0.bin
 
+# setup for BF migration
+define HAL_FRAME_TYPE 12
```

Comment:
i think it should be HAL_FRAME_TYPE_DEFAULT

Reply:
done